toolkit/scss/_browse_list.scss

```diff
@@ -12,20 +13,76 @@
   margin-bottom: $gutter;
 }
 
-.browse-list-item-link {
-  @include inline-block;
-  @include heading-27;
+.browse-list-item-link,
+a.browse-list-item-link {
+
+  display: inline;
+  @include heading-24;
   font-weight: bold;
   text-decoration: none;
+  border-bottom: 1px solid;
```

Comment:
Would be nice to add a comment to explain why there's no colour specified here so it's clear it wasn't missed.